chrony/Dockerfile

```diff
@@ -22,20 +22,20 @@ ARG BUILD_VERSION
 
 # Labels
 LABEL \
-    io.hass.name="chrony" \
-    io.hass.description="chrony NTP Server" \
+    io.hass.name="${BUILD_NAME}" \
+    io.hass.description="${BUILD_DESCRIPTION}" \
     io.hass.arch="${BUILD_ARCH}" \
     io.hass.type="addon" \
     io.hass.version=${BUILD_VERSION} \
     maintainer="Paul Sinclair <hello@addons.community>" \
-    org.opencontainers.image.title="chrony" \
-    org.opencontainers.image.description="chrony NTP Server" \
+    org.opencontainers.image.title="${BUILD_NAME}" \
```

Comment:
The ARG Definition is missing above the LABEL command.

Reply:
doh, good catch, think I probably missed that on Bookstack as well